src/components/RecipeBox.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+
+export class RecipeBox extends Component {
+	render() {
+		return (
+			<div className="recipeBox">
```

Comment:
Q: What is the encouraged convention for css? `cssClassName` or `css-class-name`?